### ISSUE TITLE
Choosing nicks

### ISSUE THREAD

**Issue by DarthGandalf** [kvirc/KVIrc#932]
**Reported by Nakre on 31 Aug 2010 20:56:31 UTC**
Well there are two things abt choose nick in the nick list
1- When u do choose fast with ur mouse the client seems doesnt choose some nicks then when u do a bit slow it choose them all.. Try choose 10+ nicks quickly..

2- in the channel when u click a nick it goes to msg box as nick: which is kinda no usefull coz we already have nick completor as nick first word n tab goes nick: 

Would be better if we click a nick in the channel the client take us on the nick at nicklist atm it doesnt and hard to find the nick in big rooms..

---

Migrated from: https://svn.kvirc.de/kvirc/ticket/932

**Comment by DarthGandalf** [comment]
**Comment by ctrlaltca on 01 Sep 2010 08:02:20**
1. It works here, i suspect it's a slowing down of the gui due to the pc/libraries/whatever (maybe if you tell us your os/distro/window manager, kvirc version etc we can try to reproduce it)
2. When you click on a nickname link kvirc already selects it. Additionally, the "OnChannelNickLinkClick" gets fired; you can implement your favourite action creating an handler for this event. The default handler does this:

```
if($0 != "") window.insertInInputText $window $0$option(stringNickCompletionPostfix);
```

---

Resolution: none → wontfix
Status: new → closed

**Comment by DarthGandalf** [comment]
**Comment by Nakre on 01 Sep 2010 17:56:38**
Im using KVIrc 4.1.1 svn-4901 'Equilibrium' 20100818 - build 2010-08-19 15:43:01 UTC - Ultimate Edition  (Build 7600) on Windows 7.

As well for the selecting thing right kvirc does select the nick at the nicklist bar what i mean is nicklist bar doesnt move up or down auto to point to the selected nick 
so that you dont have to search for it in the nicklist. 
And yep i see what the event for is.(good event btw hehe)

---

Resolution: wontfix → none
Status: closed → reopened

**Comment by DarthGandalf** [comment]
**Comment by Nakre on 01 Sep 2010 19:29:18**
Ok found out one more thing example ur on an A... nick and u push P to see the nicks start with P at the nicklist.. it moves to there but nicklist bar still at A..
So scroll shouldnt be down to P.. nicks as well ? ummm

**Comment by DarthGandalf** [comment]
**Comment by ctrlaltca on 03 Sep 2010 14:29:16**
implemented in r4973

---

Resolution: none → fixed
Status: reopened → closed

**Comment by DarthGandalf** [comment]
**Comment by Nakre on 06 Sep 2010 22:45:38**
ok no nick choosing at all from channel window atm with KVIrc-4.1.1-r4987-full
even i thought this fix may sorted the nicklist menu goes up/down to the nick u choosen to point where is the nick at the nicklist menu ?
that last one i mentioned works fine :)

---

Resolution: fixed → none
Status: closed → reopened

**Comment by DarthGandalf** [comment]
**Comment by Nakre on 06 Sep 2010 22:48:23**
for the 1- When u do choose fast with ur mouse the client seems doesnt choose some nicks then when u do a bit slow it choose them all.. Try choose 10+ nicks quickly.. 
i still have same problem not sure if anyone else tried or test this yet?

**Comment by DarthGandalf** [comment]
**Comment by Nakre on 08 Sep 2010 22:24:01**
Replying to [comment:3 Nakre]:

> Ok found out one more thing example ur on an A... nick and u push P to see the nicks start with P at the nicklist.. it moves to there but nicklist bar still at A..
> So scroll shouldnt be down to P.. nicks as well ? ummm

This works only once and if u have "z" nick as op it goes to top of nick list menu,
2nd time u push "z" should go to bottom of niclist menu to show 2nd 3rd... nicks with "z" ?

**Comment by DarthGandalf** [comment]
**Comment by ctrlaltca on 18 Sep 2010 13:09:14**
Implemented the last feature request in r5013.
Closing the ticket since it has become unredable (too many different things discussed).

---

Resolution: none → fixed
Status: reopened → closed
